src/components/toolbars/content-toolbar.tsx

```diff
@@ -70,7 +77,7 @@ const ContentToolbar = (props) => {
     );
 
     const moveElementOnError = useCallback(
-        (errorMessages, params, paramsOnErrors) => {
+        (errorMessages: string[], params: unknown, paramsOnErrors: string[]) => {
```

Comment:
Same here (useMultipleDeferredFetch is waiting for unknown[] as type of paramsOnError)